storage/Utils/Logger.h

```diff
@@ -20,29 +20,28 @@
  * find current contact information at www.novell.com.
  */
 
-
 #ifndef STORAGE_LOGGER_H
 #define STORAGE_LOGGER_H
 
-
 #include <string>
 
-
 namespace storage
 {
-
-
     enum class LogLevel { DEBUG = 0, MILESTONE = 1, WARNING = 2, ERROR = 3 };
 
-
     /**
-     * The Logger class.
+     * Base Logger class
      */
     class Logger
     {
     public:
 
-	Logger() {}
+	static Logger* create(const std::string& filename = "");
```

Comment:
So, is this a singleton class or not?

It looks weird to me to have both a `create()` factory method and a singleton `get_instance()` method. Which one is it? I am confused.